From: "Matteo F. Vescovi" <mfv@debian.org>
Date: Thu, 9 Jul 2015 22:31:02 +0200
Subject: blender_thumbnailer

* Add blender.thumbnailer file
* Change to python3 since lintian is messing around with that
* Move thumbnailer script to an appropriate destination
---
 blender.thumbnailer                | 4 ++++
 release/bin/blender-thumbnailer.py | 2 +-
 source/creator/CMakeLists.txt      | 2 +-
 3 files changed, 6 insertions(+), 2 deletions(-)
 create mode 100644 blender.thumbnailer

diff --git a/blender.thumbnailer b/blender.thumbnailer
new file mode 100644
index 0000000..05d4fae
--- /dev/null
+++ b/blender.thumbnailer
@@ -0,0 +1,4 @@
+[Thumbnailer Entry]
+TryExec=/usr/share/blender/scripts/blender-thumbnailer.py
+Exec=/usr/share/blender/scripts/blender-thumbnailer.py %i %o
+MimeType=application/x-blender;
diff --git a/release/bin/blender-thumbnailer.py b/release/bin/blender-thumbnailer.py
index d3b31d6..9685d46 100755
--- a/release/bin/blender-thumbnailer.py
+++ b/release/bin/blender-thumbnailer.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # ##### BEGIN GPL LICENSE BLOCK #####
 #
diff --git a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
index 1b2dd21..ef7b0b0 100644
--- a/source/creator/CMakeLists.txt
+++ b/source/creator/CMakeLists.txt
@@ -487,7 +487,7 @@ if(UNIX AND NOT APPLE)
 		)
 		install(
 			PROGRAMS ${CMAKE_SOURCE_DIR}/release/bin/blender-thumbnailer.py
-			DESTINATION bin
+			DESTINATION ${CMAKE_INSTALL_PREFIX}/share/blender/scripts
 		)
 		set(BLENDER_TEXT_FILES_DESTINATION share/doc/blender)
 	endif()
